ui/src/main/js/page/settings/standalone/SettingsEncryptionConfiguration.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import * as PropTypes from 'prop-types';
+import * as ConfigurationRequestBuilder from 'common/util/configurationRequestBuilder';
+import * as fieldModelUtilities from 'common/util/fieldModelUtilities';
+import PasswordInput from 'common/input/PasswordInput';
+import ConfigurationForm from 'page/channel/email/standalone/ConfigurationForm';
+import * as HttpErrorUtilities from 'common/util/httpErrorUtilities';
+import { SETTINGS_FIELD_KEYS, SETTINGS_INFO } from 'page/settings/SettingsModel';
+
+const SettingsEncryptionConfiguration = ({
+    csrfToken, errorHandler, readonly, displaySave
+}) => {
+    const encryptionRequestUrl = `${ConfigurationRequestBuilder.ENCRYPTION_API_URL}`;
+
+    const [settingsEncryptionConfig, setSettingsEncryptionConfig] = useState({});
+    const [errors, setErrors] = useState(HttpErrorUtilities.createEmptyErrorObject());
+
+    const [passwordFromApiExists, setPasswordFromApiExists] = useState(false);
```

Comment:
I imagine you'd need state for each password field. This state is meant to track whether or not a specific field has a value according to the API and since you have 2 password fields, you'll need to track them both individually

Reply:
Makes sense the email has only the auth password.  So there was only one field to use as an example.  Good to know for my own reference in the future.